packages/wonder-blocks-popover/components/popover-content-core.js

```diff
@@ -0,0 +1,120 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import Colors from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+import CloseButton from "./close-button.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * The content to render inside the popover.
+     */
+    children: React.Node,
+
+    /**
+     * Close button label for use in screen readers
+     */
+    closeButtonLabel?: string,
+
+    /**
+     * When true, the close button is shown; otherwise, the close button is not shown.
+     */
+    closeButtonVisible?: boolean,
+
+    /**
+     * Whether we should use the default light color scheme or switch to a
+     * different color scheme.
+     */
+    color: "blue" | "dark" | "light",
+
+    /**
+     * Called when the popover closes
+     */
+    onClose?: () => mixed,
+
+    /**
+     * Custom styles
+     */
+    style?: StyleType,
+
```

Comment:
maybe we should elaborate where exactly these styles will be applied

Reply:
Good call, I'll update this in PR 3/4.